docs/readmes/lte/dev_notes.md

```diff
@@ -359,58 +359,22 @@ num_enbs: 1
 
 ## Combining XML Unit test reports
 
-At the moment of writing, running the unit test targets does not generate the aggregated XML report file by default. To get the report of all unit tests, you have two options.
+When running unit tests with bazel test, report `.xml` files are automatically generated. The reports can be found at `$MAGMA_ROOT/bazel-testlogs/path/to/test/test_name/test.xml`, e.g. `$MAGMA_ROOT/bazel-testlogs/lte/gateway/python/magma/pipelined/tests/test_ifaces/test.xml`.
 
-### Option 1: Running unit test target
+### Combining individual test reports into a single report file
 
-If you prefer a simple method, you can simply do:
+In CI the individual test reports are merged with the `runtime_report.py` script.
 
-```bash
-cd ${MAGMA_ROOT}/lte/gateway/
-make test_oai_runtime
-```
-
-And you will get the combined report at `${MAGMA_ROOT}/report/merged_report/report_all_tests.xml`
-
-### Option 2: Manual method
-
-The script `runtime_report.py` is intended to use as in option 1. This section is only served as a guideline in case, in the future, a developer would like to make some changes to this script or how to use it.
-
-#### Enabling generating the unit test report
-
-Setting the value for the `$GTEST_OUTPUT` as below:
-
-```bash
-export GTEST_OUTPUT=xml:/where/you/want/to/store/the/reports/
-```
-
-#### Running the OAI unit test targets as usual on `magma` VM
-
-```bash
-make test_oai
-```
-
-The report XML files are generated inside your desired folder.
-
-#### Combine all the reports into a single report file
-
-Assume that you chose to store all the generated report files at `/where/you/stored/the/reports/` which is called the **working directory** here. You can simply set a regex pattern that matches the **relative** paths of all XML report files from the **working directory** as in the example below.
-
-```bash
-cd ${MAGMA_ROOT}/lte/gateway/
-## optional: activate your python virtual environment
-python3 python/scripts/runtime_report.py -i [regex_pattern_matches_relative_path_of_report] -w /where/you/stored/the/reports/
-```
-
-For example, when you generated all the `.xml` report files at `${MAGMA_ROOT}/logs/`  and want to combine them, you can run:
+Locally the reports can be merged by running:
 
 ```bash
-python3 python/scripts/runtime_report.py -i .+\\.xml$$ -w ${MAGMA_ROOT}/logs/
+cd ${MAGMA_ROOT}
+python3 lte/gateway/python/scripts/runtime_report.py -i [regex_pattern_matches_relative_path_of_report] -w /where/you/stored/the/reports/
```

Comment:
Is it expected that you store the reports in some other folder? I would naively think of running this command with the reports still in the bazel-testlogs folder

Reply:
Yes, they need to be copied first, clarified.